### ISSUE TITLE
[Question] Is it possible to inference on multiple GPUs?

### ISSUE THREAD

**Issue by rubydoudou** [haotian-liu/LLaVA#190]
### Question

I am encountering an issue with the 13B model inference where it fails to complete due to insufficient GPU memory. My computer has two GPUs, each with a memory of 24GB.. Inference using the 7B model is fine.

I am wondering if it is possible to perform CLI inference with two GPUs. I would appreciate any help or suggestions on how to resolve this issue. 

Thank you.

**Comment by zglin** [comment]
Pass --num-gpus <N> to the llava.serve.model_worker command for inference

**Comment by rubydoudou** [comment]
When I run this command: ' python -m llava.serve.model_worker --host 0.0.0.0 --controller http://localhost:8100 --port 8400 --worker http://localhost:8400 --model-path ./LLaVA-13B-v0 --multi-modal --num-gpus 2' ,  it returned the error below.

OutOfMemoryError: CUDA out of memory. Tried to allocate 50.00 MiB (GPU 0; 23.68 GiB total capacity; 11.04 GiB already
allocated; 1.38 MiB free; 11.04 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting
max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

I checked the GPU memory using 'watch -n 1 nvidia-smi', it seemed that only the memory of the first GPU increased. Should I change the script model_worker.py manually to make it work?
